wtxmgr/tx.go

```diff
@@ -12,11 +14,21 @@ import (
 	"github.com/Qitmeer/qitmeer/params"
 )
 
+var Coins = map[types.CoinID]string{
+	types.MEERID: "MEER",
+	types.QITID:  "QIT",
+}
+
+var CoinIDs = map[string]types.CoinID{
```

Comment:
In the future can consider using `types.NewCoinID` and `types.CoinIDList`. https://github.com/Qitmeer/qitmeer/pull/447/files#diff-ffe243d73c3535c4248478654b7efdca6901e5f0f5e1108646607ffeb2eab366R87

Reply:
ok